docs/development/pigs/pig-013.rst

```diff
@@ -0,0 +1,254 @@
+notably Astropy, but also e.g. Numpy or Scipy or Matplotlib) is very small.
+
+Required dependencies
+=====================
+
+We propose to update the Gammapy required dependencies as shown in the following
+table (the release dates for the packages are shown in parentheses are were
+obtained from https://pypi.org/).
+
+===============  ===============  ===============
+Dependency       Gammapy 0.12     Gammapy 0.14
+===============  ===============  ===============
+Python           3.5  (Sep 2015)  3.6  (Dec 2016)
+Numpy            1.10 (May 2016)  1.16 (Jan 2019)
+Scipy            0.15 (Jan 2015)  1.2  (Dec 2018)
+Astropy          2.0  (Jul 2017)  3.2  (Jun 2019)
+regions          0.3  (Sep 2018)  0.5  (Jul 2019)
+pyyaml           unclear          3.13 (Jul 2018)
+click            unclear          7.0  (Sep 2018)
+===============  ===============  ===============
+
+Notes:
+
+- We already mentioned the possibility to drop Python 3.5 support in
+  :ref:`pig-003`. One reason is that Anaconda and conda-forge (our main
+  distribution channel, used in our testing continuous integration setup) only
+  contains Python 2.7, 3.6 and 3.7 at this point (and 3.8 added in fall 2018),
+  i.e. testing on Python 3.5 is already extra effort. Also, Python 3.6 allows us
+  to use f-strings and just dict instead of a mix of dict and OrderedDict like
+  we have now, it's nice for developers. E.g. Sunpy or ctapipe already require
+  Python 3.6 or later.
+- This update is to very recent versions for dependencies. We could keep support
+  for older versions, e.g. Numpy 1.13 (Jun 2017), Scipy 1.0  (Oct 2017) and
+  Astropy 3.0 (Feb 2018) or even older. But it wasn't clear to us if doing so
+  would be of use to anyone, so we defaulted to this propsal.
+- For Astropy, we could keep support for version 2.0, but there are many small
+  new features, fixes and performance improvements in 3.0, 3.1 and 3.2 (see
+  `Astropy whatsnew`_) that will make our life easier: much improved
+  Quantity-support for functions in Numpy and Scipy, support for the FITS time
+  standard and better support for Time columns in Table, the new
+  ``astropy.uncertainty`` package which we could use (see :ref:`pig-14`_), the
+  ``SkyCoord.directional_offset_by`` method for field of view coordinates,
+  common WCS API, the new ``astropy.timeseries``, ASDF, ...
+- A major motivation to update to very recent versions is
+  that the ``regions`` package is still under development (see :ref:`pig-010`_).
+  In Gammapy 0.13 we require regions 0.4, and we plan to make a regions 0.5
+  release with further features and fixes in July, and to require that for
+  Gammapy 0.14.
+- For ``pyyaml`` and ``click`` we never had a version specified so far or a
+  version-dependent issue reported, i.e. any version from recent years worked.
+  Still, to have well-defined testing and support, we choose minimum required
+  versions from 2018 here.
+
+Optional dependencies
+=====================
+
```

Comment:
May be put a short definition of "optional dependency" here.

Reply:
will do